### DISCUSSION TITLE
Conversion from GFE Save issue, directory problem?

### DISCUSSION THREAD

**Discussion by stgreenb** [LizardByte/community#109] [Sunshine] [answered]
I am starting to migrate from GFE to sunshine (since they are going to deprecate the feature).  Just doing some testing, and I am having an issue in that one windows game when launched doesn't see the previous save files. I have a suspicion that this is due to the start/running directory or maybe user identity (but I'm open to other ideas) . Opening the game natively in windows, or via GFE, the save is seen, opening in moonlight via sunshine and its not.  Adjusting the working directory to match the start in directory does not fix the issue (see images of windows shortcut, sunshine and gfe). 

![image](https://user-images.githubusercontent.com/18483964/208307505-d092f5bf-1bb0-4bf8-ba9b-2500e019dd30.png)
![image](https://user-images.githubusercontent.com/18483964/208307943-56e07327-032e-4a33-abb2-05bdc98d909f.png)

**Comment by ReenigneArcher** [comment, marked as answer]
I would suggest disabling the sunshine service for now. I'm working on some updates to the docs. These are WIP, but it's somewhat explained here: https://lizardbyte--574.org.readthedocs.build/projects/sunshine/en/574/about/usage.html#application-list

**Reply by stgreenb** [reply]
Yes this solved.

**Reply by mlbrnm** [reply]
Would doing this be a workaround for this issue? Thanks!

<img width="310" alt="image" src="https://user-images.githubusercontent.com/114309689/208528215-6165fc48-7d19-429c-ad7c-6b4c01f5f8bc.png">

Or alternatively, just launching "desktop" from Moonlight and then running the game that way?

EDIT: It seems changing the service log on like that completely broke the service functionality, so I guess not!
